roles/osasinfra/tasks/test_shiftstack.yml

```diff
@@ -0,0 +1,78 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Clone the repository '{{ cifmw_osasinfra_shiftstack_qa_repo }}'
+  ansible.builtin.include_tasks: tasks/exec_command_in_pod.yml
+  vars:
+    namespace: "{{ cifmw_osasinfra_client_pod_namespace }}"
+    pod: "{{ cifmw_osasinfra_client_pod_name }}"
+    command: "git clone {{ cifmw_osasinfra_shiftstack_qa_repo }}"
+    log_file_name: "clone_shiftstack_qa_repo.log"
+
+- name: Install the ansible collection requirements
+  ansible.builtin.include_tasks: tasks/exec_command_in_pod.yml
+  vars:
+    namespace: "{{ cifmw_osasinfra_client_pod_namespace }}"
+    pod: "{{ cifmw_osasinfra_client_pod_name }}"
+    command: "cd shiftstack-qa && ansible-galaxy collection install -f -r requirements.yaml"
+    log_file_name: "install_requirements.log"
+
+#TODO: check the "{{ cifmw_run_test_osasinfra_testconfig }}" exists
+
+- name: OCP install and test block
+  block:
+    - name: Test Openshift on Openstack with '{{ cifmw_run_test_osasinfra_testconfig }}' test configuration
+      ansible.builtin.include_tasks: tasks/exec_command_in_pod.yml
+      vars:
+        namespace: "{{ cifmw_osasinfra_client_pod_namespace }}"
+        pod: "{{ cifmw_osasinfra_client_pod_name }}"
+        command: "cd shiftstack-qa && ansible-navigator run playbooks/ocp_testing.yaml -e @jobs_definitions/{{ cifmw_run_test_osasinfra_testconfig }}"
+        log_file_name: "ocp_testing.log"
+
+  rescue:
```

Comment:
It's observed on https://sf.hosted.upshift.rdu2.redhat.com/zuul/t/tripleo-ci-internal/build/ff99fdbfeb6b47b28145b520b33d23e4 (where the shiftstack installation failed) that the job is TIMING OUT and not FAILING. 

Do you know what is that happening? Should we create a task to investigate the error reported? We should keep into consideration also the UNSTABLE: https://github.com/shiftstack/shiftstack-qa/blob/main/collection/tools/roles/tools_stage_results/tasks/mark_stage_unstable.yml

Reply:
you should fail in rescue